### DISCUSSION TITLE
Doing adult things…

### DISCUSSION THREAD

**Discussion by Codectory** [Codectory/AutoActions#141] [General]
I want to give you a small update as I see the issues are increasing. 
Two years ago, I had to pause this project for some weeks because we got a little daughter and didn't had enough time. Since last year, we got a little house and we are doing most of the work ourselves. So, there is even less time for anything beside my family, my job and the house.  
I just want to let you know, that this project is not dead. 
I hope I will restart at some point this year.

**Comment by Fr0stX76** [comment]
First congrats on the baby and the new house!

And thanks for this amazing program. Not going to lie, this has been a game changer as it has added so much more flexibility to Windows once I started pairing it with scripts! I can run Gsync on/off, change resolution/refresh rate, change CPU profile, change GPU profile.. All that base on the game/app I use.

All that to say, this software is a must for me as it is and I appreciate the work you already put in it. Take the time you need and I will be looking forward new update when you are ready to resume once life allows.

Again, thanks!

**Reply by Nama** [reply]
I'm curious about how you change CPU and GPU profile. Could you tell us please?

For CPU, you probably mean the power plan. I even wrote [something for that](https://github.com/Nama/Windows-PowerPlan-Switcher/), but using a [modified power plan](https://www.computerbase.de/forum/threads/energiesparplan-zen2-zen3-ryzen-3000-5000.1934824/) for ryzens makes that obsolete.

For GPU, I used Nvidia Inspectors MultiMonitorPowerSaver. Didn't have any effect anymore on my RTX 2070, so I dropped that, too.

**Reply by Fr0stX76** [reply]
@Nama 

Sorry for the (very) late answer. For CPU I use a script that calls for the hotkeys linked to throttlestop. But this will be irrelevent to you since Throttlestop is an Intel thing and you are on AMD.

As for GPU, I so wish I could still use MultiMonitorPowerSaver! That thing was amazing. But sadly it does not work with 3000 and 4000 GPUs. It will just crash my PC with a blackscreen whenever I change resolution or refresh rate...

My point was:  Once you start combining AutoActions with Autohotkeys, it become a game changer (pun intented) to any issue when running a game, I can usually come up with some script that will lauch with the game.

For exemple, I have Gsync turn on automatically by using the exe from https://github.com/kwizatz88/GsyncSwitch . I made a script that lauches the exe with a 100ms delay. Then I run the same script upon game exit. This way I don`t have Gsync running all the time (which is best for me on desktop or when watching movies). Add to this that I can have Windows HDR turn on, CPU profile go max, change RTSS OSD profile per game (using a script w/ RTSS hotkeys). And so on.

I just hope Codectory find some time to update. One day, I will have to switch to W11 and I`m kinda scared AutoActions will not work anymore, or not as well...

**Comment by hoonlight** [comment]
congratulations!

**Comment by SullieLore** [comment]
Thanks for the great app and hope you're doing well with the changes in your life.

**Comment by semtexxl** [comment]
Well it seems that autoactions are no longer working under Win 11 24H2. Is there any alternative guys?

**Reply by Fr0stX76** [reply]
Really? Dang that sucks big time. Not going to upgrade to 24H2 then. Hoping either the dev comes back, or that someone create a fork

**Reply by semtexxl** [reply]
Ok I found out it is working. You have to disable ACM (Auto Color Management) option.

**Reply by Fr0stX76** [reply]
Oh. That is great news! Thanks for following-up.

**Reply by Codectory** [reply]
Should be fixed now, regardsless of ACM
